Modules/posixmodule.c

```diff
@@ -7625,6 +7625,9 @@ static void warn_about_fork_with_threads(const char* name) {
             num_python_threads = atoi(field);  // 0 on error
         }
     }
+    // XXX This counts the number of system threads, but Python code can only
+    // call threading.Thread.join(), which can return before the system thread ended.
+    // This function could therefore print a spurious warning in unlucky cases.
```

Comment:
It's not spurious though, the existence of a thread doing anything is accurately identified as a potential problem. The annoyance that led to this though is that Python didn't provide a concrete way to guarantee a thread has exited.